doc/nrf/protocols/lte/index.rst

```diff
@@ -0,0 +1,214 @@
+
+The following is a LTE bandwidth comparison for LTE-M and NB-IoT:
+
++----------------------------------+----------------------------------+----------------------------------+
+|                                  | LTE-M                            | NB-IoT                           |
++==================================+==================================+==================================+
+| Bandwidth                        | Less bandwidth (1.4 MHz)         | Less bandwidth (200KHz)          |
++----------------------------------+----------------------------------+----------------------------------+
+| Modulation Techniques            | No 64QAM                         | No 64QAM or 16QAM                |
++----------------------------------+----------------------------------+----------------------------------+
+| Antenna Techniques               | No MIMO                          | No MIMO                          |
++----------------------------------+----------------------------------+----------------------------------+
+| OFDM subcarriers                 | Less OFDM subcarriers (72)       | OFDM subcarriers (12)            |
++----------------------------------+----------------------------------+----------------------------------+
+| Duplexing                        | Can use half-duplex              | Must use half-duplex             |
++----------------------------------+----------------------------------+----------------------------------+
+| Modulation in :term:`Uplink (UL)`| QPSK or 16QAM in both UL and DL  | BPSK or QPSK in UL depending on  |
+| and :term:`Downlink (DL)`        | depending on signal quality      | signal quality.                  |
+|                                  |                                  | Only QPSK in DL                  |
++----------------------------------+----------------------------------+----------------------------------+
+
+Location support
+================
+
+A factor governing the usage of the protocol is the support for the technologies in the areas where you wish to install your product.
+Most cellular operators support both LTE-M and NB-IoT, but there are geographic locations where only one of these is supported.
+See `Mobile IoT Deployment Map`_ for more information.
+
+Certifications
+==============
+
+Some MNOs have their own certifications, like Verizon and Vodafone.
+But many operators only require GCF (Global Certification Forum) and PTCRB certifications that devices work as per standard, and regulatory certifications.
+
+For more information about certification related to the nRF91 Series, refer to the following pages:
+
+* `nRF91 Series certifications`_
+* `Certifying a Cellular IoT Device`_
+
+SIM card support
+================
+
+SIMs that are used with the nRF91 Series devices must have low-power WAN support and must be supported by the MNO.
+For more information on the iBasis SIM compatible with the nRF91 Series devices, refer to the `iBasis IoT network coverage`_ page.
```

Comment:
```suggestion
For more information on the iBasis SIM bundled with the nRF91 Series devices, refer to the `iBasis IoT network coverage`_ page.
```

Reply:
Ibasis is only bundled for nRF9160 devices.

Onomondo SIMs are bundled for nRF9161 DK.
lte-m https://onomondo.com/product/coverage/lte-m-networks/
nb-iot https://onomondo.com/product/coverage/nb-iot-networks/